src/Handler/Mapping/MethodName/HandleClassNameWithoutSuffix.php

```diff
@@ -34,11 +36,12 @@ public function __construct(string $suffix = 'Command')
     {
         $this->suffix       = $suffix;
         $this->suffixLength = strlen($suffix);
+        $this->handleLastPartOfClassName = new HandleLastPartOfClassName();
     }
 
     public function getMethodName(string $commandClassName) : string
     {
-        $methodName = parent::getMethodName($commandClassName);
+        $methodName = $this->handleLastPartOfClassName->getMethodName($commandClassName);
```

Comment:
I normally avoid constructing anything in the constructor but in this case I'll allow it. :ok_woman: :heavy_check_mark: